radicle-cli/examples/rad-remote.md

```diff
@@ -0,0 +1,23 @@
+Now lets add a bob as a new remote
+
+```
+$ rad remote add did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk --alias bob
+❲🚀❳ Remote bob added with rad://z42hL2jL4XNk6K8oHQaSWfMgCL7ji/z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk
+```
+
+Now, we can see that there is a new new remote in the list of remotes
+
+```
+$ rad remote list
+bob
+rad
```

Comment:
We should show the DIDs as well here

Reply:
Maybe it is better to show directly the remote in the table?

This is my local code

```
➜  heartwood git:(master) rad remote    
 fetch  rad                                              rad://z3gqcJUoA1n9HaHKufZs5FCSG…
 fetch  z6MksFqXN3Yhqk8pTJdUGLwATkRfQvwZXPqR2qMEhbS9wzpT rad://z3gqcJUoA1n9HaHKufZs5FCSG…

```

The idea here is to show the remote in the table, and in the user wants another view can always see the `git remote -v` I see no reason so simulate  `git` here

What do you think?

Reply:
I implemented it in https://github.com/radicle-dev/heartwood/pull/524/commits/57ca2401427b54cace0dd20b3b883c2898764a20

happy to iterate over it.

I think this will deserve a use like `rad patch list`

Reply:
I couldn't get `rad remote -v`s output to work with the example testing.  Not sure where the problem was coming in.

Reply:
@slack-coder with the commit https://github.com/radicle-dev/heartwood/commit/57ca2401427b54cace0dd20b3b883c2898764a20 there is no longer the verbose flag, I think with a table there is no reason of having a verbose flag? 

What do you think?